### ISSUE TITLE
Duplicated messages

### ISSUE THREAD

**Issue by luksol** [andyholmes/gnome-shell-extension-gsconnect#76]
Hi. Whenever I am receiving SMS message it is bering duplicated 4 times, so I am getting 5 identical messages. It has happened recently after an update to the new KDE Connect version on my mobile, any idea how to fix/debug this?

**Comment by andyholmes** [comment]
Yep, this happened to me too. If you were to check the debug log you'd see they are being duplicated on the Android side, but no need for that.

I believe I fixed this by doing a force close on the app, then clearing cache (but no need to clear all data), then reopening the app and it was fixed :)

Let me know if that doesn't fix it for you.
